lib/components/molecules/QuoteMedia/QuoteMedia.js

```diff
@@ -28,16 +26,14 @@ const QuoteMedia = ({
       {title}
     </Heading>
     <div className={children && quoteProp ? 'col-1-2' : 'col-1'}>
```

Comment:
Why are you testing for quoteProp when you don't even use it in your component. Make this 'col-1-2' : 'col-1' thing explicit variation driven

Reply:
fixed | working on: Make this 'col-1-2' : 'col-1' thing explicit variation driven